config/manager/manager.yaml

```diff
@@ -56,6 +56,8 @@ spec:
       containers:
       - name: ibmcloud-machine-controller
         image: quay.io/cluster-api-provider-ibmcloud/controller:latest
+        args:
+          - --wait_transactions_timeout=600s
```

Comment:
It seems we do not need to specify command line options with default value. Any consideration here?

Reply:
Because no doc to tell user the parameter name, user can learn it by review the yaml file. 
I thought it may help to avoid user input wrong parameter name, and no any impact.

Reply:
We have `-h` to show the parameter and the help message. I think leave it here is fine for now. When we document it later, we can remove this.

Reply:
OK